chemclipse/plugins/org.eclipse.chemclipse.xxd.converter.supplier.chemclipse/src/org/eclipse/chemclipse/xxd/converter/supplier/chemclipse/Activator.java

```diff
@@ -29,30 +27,17 @@ public static BundleContext getContext() {
 	 * (non-Javadoc)
 	 * @see org.osgi.framework.BundleActivator#start(org.osgi.framework.BundleContext)
 	 */
+	@Override
 	public void start(BundleContext bundleContext) throws Exception {
 
 		Activator.context = bundleContext;
-		//
```

Comment:
I would like to keep this part as it shows up in the console when testing the application. It's a string hint for the developers to take care of the *.ocb format. As we can't enforce more strict permissions on the *.ocb bundle, the message shall ensure that no modification happens that is not reviewed and authorized.

Reply:
I debug a lot via the console. To me, it is just visual clutter and noise. It is displayed and scrolled away fast. I doubt there will be a sudden influx of developers making changes and even if so GitHub code owners protection does it way more effectively.

Reply:
OK, let's try it with the CODEOWNERS directive.